legacy/cli/audit.go

```diff
@@ -56,6 +56,8 @@ func RunAuditCmd(opts *AuditOptions) error {
 	}
 
 	if opts.Verbose {
+		// Enable verbose logging.
+		logrus.SetLevel(logrus.DebugLevel)
```

Comment:
We should now also probably delete the useless Debug wrapper we have around logrus.Debug (because the if-check is redundant now).

Reply:
In that case, we don't even need [VerboseLogging](https://github.com/kubernetes-sigs/k8s-container-image-promoter/blob/master/legacy/audit/types.go#L46) in ServerContext.